src/paperless/settings.py

```diff
@@ -1115,6 +1116,11 @@ def default_threads_per_worker(task_workers) -> int:
 # Specify the default date order (for autodetected dates)
 DATE_ORDER = os.getenv("PAPERLESS_DATE_ORDER", "DMY")
 FILENAME_DATE_ORDER = os.getenv("PAPERLESS_FILENAME_DATE_ORDER")
+DATE_PARSER_LANGUAGES = list(
+    LocaleDataLoader().get_locale_map(
+        languages=json.loads(os.getenv("PAPERLESS_DATE_PARSER_LANGUAGES", "[]")),
+    ),
+)
```

Comment:
This does seem like an easy win with not a lot of trade-off. That said, making it 100% opt-in seems like a missed opportunity. I think it's worth making this default to the OCR languages while still allowing for manual control. I understand they are not the same, but I suspect we can interpolate between them without too much misery (famous last words?)

Reply:
Pycountry is a pretty heavy dependent for just one thing.  I think the idea was more a manual interpolation, similar to how nltk is configured with some conversions between formats

Reply:
Yea I’d agree. Don’t want to make you chase your tail but was thinking given how small the sets are here a kind of manual conversion would be feasible. Of course I haven’t done that so perhaps I’m wrong. But I don’t think it’s “worth” a new dep. let us know if you’re up for the challenge (or not)!